docs.Dockerfile

```diff
@@ -12,5 +12,9 @@ COPY . .
 RUN npm run build-docs
 
 # Serve with nginx
-FROM ghcr.io/kibalabs/app-serve:latest
+FROM nginx:1.16.0-alpine
```

Comment:
this seems wierd - why did this not work with the other image?

Reply:
I am not sure, but I have also not seen the nginx config in the `app-serve` image, I took this config from an youtube video.